### PR TITLE
Task1 unit tests and functionality

### PR DESCRIPTION
**Pull request by GDP254**
Please review my unit tests for task 1.

### CODE REVIEW

File: tests/print_tests.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+		self.assertEquals(output, expected_output)
+
+	def test_print_existing_unallocated_to_screen(self):
+		self.clear_stores()
+		office = Office("NDO4")
+		Dojo.add_room(office)
+		fellow = Fellow("Xone4", "Ndobile4", "0856443354", "y")
+		fellow.register()
+		office.allocate_to(fellow)
+		staff = Staff("Xone3", "Ndobile3", "0856443344", "y")
```
Comment (mnjaggah) [inline tests/print_tests.py]: you've included accommodation option for the staff object, while it's not meant to have one

Comment (GDP254) [reply tests/print_tests.py]: Noted. Thanks. will make the change soon.

File: controller.py [hunk 2]
```diff
@@ -9,6 +9,45 @@
...
+				print("Unallocated persons in the default output/File.txt")
+		else:
+			print(output)
+	else:
+		print("There are no unallocated persons to show")
+
+def print_allocations(out, file_name):
+	allocations = Room.all_allocations()
+	output = Room.members(allocations, room_tag=True)
+	if len(output) > 0:
```
Comment (mnjaggah) [inline controller.py:30]: a better expression to check a list is not empty would be  if output:

**Review by mnjaggah — COMMENTED** [review]
method definition look good to me. I like how you defined separate methods for allocations outside the print methods

**Review by maquchizi — APPROVED** [review]
Looking good. Please merge and continue to next task.

**Pull request merged**